### PR TITLE
#81: Updated poetry and udf mock

### LINKED ISSUE #81: Update poetry version and udf-mock

### PR DESCRIPTION
**Pull request by MarleneKress79789**
fixes #81

### CODE REVIEW

File: scripts/build/poetry_utils.sh [hunk 1]
```diff
@@ -8,9 +8,9 @@ init_poetry () {
 }
 
 request_install_poetry () {
-  install_command="cat get-poetry.py | python3 -"
-  curl -sSL -o get-poetry.py https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py
-  sed -i 's/allowed_executables = \["python", "python3"\]/allowed_executables = ["python3", "python"]/'  get-poetry.py 
+  install_command="cat install-python-poetry.py | python3 -"
+  curl -sSL -o install-python-poetry.py https://install.python-poetry.org
+  sed -i 's/allowed_executables = \["python", "python3"\]/allowed_executables = ["python3", "python"]/'  install-python-poetry.py
```
Comment (tkilias) [inline scripts/build/poetry_utils.sh:13]: this is not necessary anymore, they fixed this with the new installer

**Review by tkilias — CHANGES_REQUESTED** [review]

File: scripts/build/poetry_utils.sh (hunk 1 above)
Comment (MarleneKress79789) [reply scripts/build/poetry_utils.sh:13]: Which part are you talking about? just the line 13 or the whole function checking weather the install worked?

Comment (tkilias) [reply scripts/build/poetry_utils.sh:13]: the sed part

Comment (MarleneKress79789) [reply scripts/build/poetry_utils.sh]: removed

**Review by tkilias — COMMENTED** [review]
Could you also mark setup.py and poetry.lock as generated.

**Review by tkilias — APPROVED** [review]

**Pull request merged**